public/app/features/dashboard-scene/scene/layout-rows/RowItemRenderer.tsx

```diff
@@ -6,6 +6,7 @@ import { GrafanaTheme2 } from '@grafana/data';
 import { selectors } from '@grafana/e2e-selectors';
 import { SceneComponentProps } from '@grafana/scenes';
 import { clearButtonStyles, Icon, Tooltip, useElementSelection, usePointerDistance, useStyles2 } from '@grafana/ui';
+import { GRID_CELL_HEIGHT } from 'app/core/constants';
```

Comment:
looks like this import isn't used.  would be good to add another constant there GRID_ROW_MIN_HEIGHT ( or whatever makes sense here )